storage/statedb/database.go

```diff
@@ -335,6 +339,17 @@ func NewDatabaseWithCache(diskDB database.DBManager, cacheSize int, daBlockNum u
 	}
 }
 
+func getCacheSize() int {
+	totalPhysicalMem := float32(common.TotalPhysicalMemGB)
+	var cacheSizeGB float32
+	if totalPhysicalMem/4 < 2.5 {
```

Comment:
How did you get 2.5?

Reply:
NewCache will take 0.75 in maximum and 1.75GB for margin.
So, Bigcache will take (total size -  2.5GB).

It would be better to shrink the size 2.5.
I will go through test and see if it is okay to shrik.

Reply:
@winnie-byun You better write the rationale behind this equation as a code comment for other developers.